packages/opencensus-instrumentation-grpc/test/test-grpc.ts

```diff
@@ -254,8 +254,12 @@ class RootSpanVerifier implements SpanEventListener {
   endedRootSpans: Span[] = [];
 
   onStartSpan(span: Span): void {}
-  onEndSpan(root: Span) {
-    this.endedRootSpans.push(root);
+  onEndSpan(span: Span) {
+    // TODO(hekike): Tests depends on the order of recorded spans.
```

Comment:
If possible, please create an issue to track this later.

Reply:
Good idea, added: https://github.com/census-instrumentation/opencensus-node/issues/501